app/javascript/registration/grid.js

```diff
@@ -5,21 +5,14 @@ import 'jquery-ui/ui/widgets/dialog'
 import 'jquery-ui/ui/widgets/progressbar'
 
 var gridContext = function() {
-  var druidFormatter = function(val, opts, rowObj) {
```

Comment:
This block was dead code, the real "druidFormatter" is defined farther down in the file.